src/Components/Layout/LayoutComponents/Header.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react'
+import { Button } from 'primereact/button'
+
+const Header: React.FC = () => {
+  const [expanded, setExpanded] = useState(false)
+
+  const handleMenu = () => {
+    setExpanded(!expanded)
+  }
+
+  return (
+    <>
+      <div
+        style={{
+          backgroundColor: 'black',
+          padding: '8px 16px',
+          display: 'flex',
+          justifyContent: 'space-between',
+          alignItems: 'center',
+        }}>
+        <img alt="Logo" src={'/assets/images/Moorfind.png'} style={{ width: 100, height: 80 }} />
+        <div style={{ display: 'flex', alignItems: 'center', gap: '8px' }}>
+          {/* <Avatar src="" /> */}
```

Comment:
remove comment

Reply:
this comment actually we need in future .